CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ changes in the following format: PR #1234***
 - BVL Feedback widget only shows notifications for the users sites / projects (PR #7848)
 - Add Date status change value in session table (PR #8350)
 - Fixed the Candidate Age at Death field label and Data Dictionary item for LINST instruments (PR #8362)
+- Allow clearing a previously entered consent status in candidate parameters (PR #7772)
+- Add code sanitizer before dangerouslySetInnerHTML is used in login to protect against XSS attacks (PR #7491)
```

Comment:
@driusan do you know if the dangerouslySetInnerHTML could cause some html to be displayed on the front end now? does it need an entry in "notes for existing projects"

Reply:
It was already using dangerouslySetInnerHTML. It's now still doing that but running it through DOMPurify first.

https://github.com/aces/Loris/pull/7491/files#diff-3ca0f06ecf3edb67af7297ed3f6034843ee1f0e420ab037369c8dab21d90d28cL180